doc/rtd/topics/faq.rst

```diff
@@ -151,6 +151,36 @@ provided to the system:
 As launching instances in the cloud can cost money and take a bit longer,
 sometimes it is easier to launch instances locally using Multipass or LXD:
 
+Why did cloud-init never complete?
+==================================
+
+To check if cloud-init is running still, run:
+```
+cloud-init status
+```
+
+To wait for clous-init to complete, run:
+```
+cloud-init status --wait
+```
+
+There are a number of reasons that cloud-init might "never complete. This list
```

Comment:
Did you mean to say "never" (in quotes) ? I don't think the quotes are necessary.